jekyll/_posts/2017-05-25-faster-command-line-tools-in-nim.md

```diff
@@ -24,7 +24,7 @@ This is quite a simple program. All that we need to do is read a file, parse it
 Luckily, Nim's standard library has many useful modules to help with common tasks like these:
 
 - The [streams](https://nim-lang.org/docs/streams.html) module provides a `FileStream` type for reading from and writing to files.
-- The [parsecv](https://nim-lang.org/docs/parsecsv.html) module provides a simple high performance CSV parser.
+- The [parsecsv](https://nim-lang.org/docs/parsecsv.html) module provides a simple high performance CSV parser.
```

Comment:
I made this PR to fix that from the very start. So @dom96 you can close this PR and @euantorano I let you refactor your article with this fix.